src/main/groovy/org/checkerframework/gradle/plugin/CheckerFrameworkPlugin.groovy

```diff
@@ -116,7 +118,8 @@ final class CheckerFrameworkPlugin implements Plugin<Project> {
       [name: "${ANNOTATED_JDK_CONFIGURATION}", descripion: "${ANNOTATED_JDK_CONFIGURATION_DESCRIPTION}"]: "org.checkerframework:${jdkVersion}:${LIBRARY_VERSION}",
       [name: "${CONFIGURATION}", descripion: "${ANNOTATED_JDK_CONFIGURATION_DESCRIPTION}"]              : "${CHECKER_DEPENDENCY}",
       [name: "${JAVA_COMPILE_CONFIGURATION}", descripion: "${CONFIGURATION_DESCRIPTION}"]               : "${CHECKER_QUAL_DEPENDENCY}",
-      [name: "${TEST_COMPILE_CONFIGURATION}", descripion: "${CONFIGURATION_DESCRIPTION}"]               : "${CHECKER_QUAL_DEPENDENCY}"
+      [name: "${TEST_COMPILE_CONFIGURATION}", descripion: "${CONFIGURATION_DESCRIPTION}"]               : "${CHECKER_QUAL_DEPENDENCY}",
+      [name: "errorProneJavac", descripion: "the Error Prone Javac compiler"]                           : "com.google.errorprone:javac:9-dev-r3297-1"
```

Comment:
Can we use version `9+181-r4173-1` here? It is [the version used by checker-framework](https://github.com/eisop/checker-framework/blob/efcd09c0bdb88e4847c33806e463e4c9c9861a26/build.gradle#L34).

Reply:
Sure. I copied that version number from your draft (https://github.com/kelloggm/checkerframework-gradle-plugin/pull/18/commits/49b6915c6d2e4df51620326ab25b519464b726eb).